packages/samples/common-types/openapi/v3/types.json

```diff
@@ -174,8 +174,7 @@
           "items": {
             "$ref": "#/definitions/ErrorAdditionalInfo"
           },
-          "readOnly": true,
-          "x-ms-identifiers": []
+          "readOnly": true
```

Comment:
How will this impact common-types generation?  Do we need to add the `@identifiers` decorator to this type?